doc/user/content/get-started/isolation-level.md

```diff
@@ -5,6 +5,10 @@ aliases:
     - /sql/consistency
     - /sql/isolation-level
     - /overview/isolation-level/
+menu:
+  main:
+    parent: get-started
+    weight: 60
```

Comment:
Also, mostly left the page in tact, other than easy fixes for some wonkiness, for this iteration.